le-taxi-api-tests/config/configs.ts

```diff
@@ -92,7 +92,10 @@ export class Configs {
       rootApiKey: this.cache.get<string>('apiTests.rootApiKey'),
       rootPath: this.cache.get<string>('apiTests.rootPath'),
       user: this.cache.get<string>('apiTests.user'),
-      password: this.cache.get<string>('apiTests.password')
+      password: this.cache.get<string>('apiTests.password'),
+      performanceTests: {
+        rootApiKey: this.cache.get<string>('apiTests.performanceTests.rootApiKey')
```

Comment:
I understand that we need to run performance test AND integration test in the same environment ?

If not, we already handle this case by having local-acceptation.yaml and local-development.yaml files as suggested by `config/local-template.yaml` file